model-optimizer/extensions/ops/RandomUniform.py

```diff
@@ -13,13 +13,21 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 """
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+from mo.front.common.partial_infer.random_uniform import tf_random_uniform_infer
 
-from mo.front.common.partial_infer.concat import concat_infer
```

Comment:
Add one more empty line before the class

Reply:
done